testproject/chtest/consumers.py

```diff
@@ -2,5 +2,6 @@
 
 
 def ws_message(message):
-    "Echoes messages back to the client"
+    """Echoes messages back to the client."""
```

Comment:
Why was this changed? There's no need for `"""` unless you're doing multiple lines.